Sport1Player/Sport1PlayerAdapter.m

```diff
@@ -227,9 +228,13 @@ -(void)shouldPresentPinFor:(NSObject <ZPPlayable>*)currentPlayableItem container
         return;
     }
     
-    NSNumber *ageRating = trackingInfo[kAgeRatingKey];
-    
-    if (ageRating.intValue >= kWatershedAge) {
+    NSString *ageString = trackingInfo[kFSKKey];
+    int ageRating = 0;
+    if (ageString.length > 0) {
+        ageRating = [ageString componentsSeparatedByString:@" "][1].intValue;
```

Comment:
This could crash if they send a string whitoush spaces.

Reply:
This could cash if they send a string just with spaces.
You should call [ageString componentsSeparatedByString:@" "] an check that the count is 2 or more.